app/src/main/java/com/github/se/icebreakrr/MainActivity.kt

```diff
@@ -47,33 +48,37 @@ import com.github.se.icebreakrr.utils.NetworkUtils
 import com.github.se.icebreakrr.utils.PermissionManager
 import com.google.android.gms.location.FusedLocationProviderClient
 import com.google.android.gms.location.LocationServices
-import com.google.firebase.Firebase
 import com.google.firebase.FirebaseApp
 import com.google.firebase.auth.FirebaseAuth
+import com.google.firebase.firestore.FirebaseFirestore
 import com.google.firebase.firestore.firestore
 import com.google.firebase.functions.FirebaseFunctions
+import dagger.hilt.android.AndroidEntryPoint
+import javax.inject.Inject
 
+@AndroidEntryPoint
 class MainActivity : ComponentActivity() {
-  private lateinit var auth: FirebaseAuth
-  private lateinit var functions: FirebaseFunctions
+  @Inject lateinit var auth: FirebaseAuth
+  @Inject lateinit var firestore: FirebaseFirestore
+  @Inject lateinit var authStateListener: FirebaseAuth.AuthStateListener
   private lateinit var locationViewModel: LocationViewModel
   private lateinit var locationService: LocationService
   private lateinit var locationRepositoryFirestore: LocationRepositoryFirestore
   private lateinit var permissionManager: PermissionManager
-  private lateinit var authStateListener: FirebaseAuth.AuthStateListener
   private lateinit var fusedLocationClient: FusedLocationProviderClient
   private lateinit var appDataStore: AppDataStore
+  private lateinit var functions: FirebaseFunctions
 
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
     AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_NO)
+    Log.d("TESTEST", "auth : $auth")
```

Comment:
`Log.d("TESTEST", "auth : $auth")`
debug: if not needed, remove logging statement